pages/profile-edit/form/fields.js

```diff
@@ -33,58 +30,53 @@ module.exports = function(showLongBio) {
     },
     user_bio: {
       type: `textarea`,
-      label: (
-        <div>
-          Bio <em>(do not include any personal information)</em>
-        </div>
-      ),
+      label: `Bio`,
       placeholder: `About you`,
+      labelClassname: `h5-heading bio`,
       fieldClassname: `form-control`,
       validator: [validator.maxLengthValidator(140)],
       charLimit: 140,
       charLimitText: function(charCount, charLimit) {
         return charLimit - charCount;
       }
     },
-    twitter: {
+    Twitter: {
```

Comment:
This changes the field's `name` attribute from `twitter` to `Twitter`. Are we doing this due to API changes? If not, this should not be changed.

Reply:
Okay, that's fine. But, how is the `.twitter` class added on this element?